site-manage/datemanager/impl/src/java/org/sakaiproject/datemanager/impl/DateManagerServiceImpl.java

```diff
@@ -1997,11 +1997,13 @@ public boolean isChanged(String toolId, String[] columns) {
 				while (i < announcements.size() && !changed) {
 					AnnouncementMessage announcement = (AnnouncementMessage) announcements.get(i);
 					if (announcement.getId().equals(id)) {
-						if (announcement.getProperties().getProperty(AnnouncementService.RELEASE_DATE) != null) {
+						boolean releaseDateActive = announcement.getProperties().getProperty(AnnouncementService.RELEASE_DATE) != null;
+						boolean retractDateActive = announcement.getProperties().getProperty(AnnouncementService.RETRACT_DATE) != null;
+						if (releaseDateActive) {
 							changed = changed || this.compareDates(Date.from(announcement.getProperties().getInstantProperty(AnnouncementService.RELEASE_DATE)), columns[2]);
```

Comment:
changed = changed || this.compareDates(Date.from(releaseDateActive ), columns[2]); ?